src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditionsImpl.hpp

```diff
@@ -0,0 +1,1447 @@
+    // Constraint damping parameter gamma2
+    ::Tags::TempScalar<26, DataVector>,
+    // Preallocated memory to store boundary conditions
+    ::Tags::Tempaa<27, VolumeDim, Frame::Inertial, DataVector>,
+    ::Tags::Tempiaa<28, VolumeDim, Frame::Inertial, DataVector>,
+    ::Tags::Tempaa<29, VolumeDim, Frame::Inertial, DataVector>,
+    ::Tags::Tempaa<30, VolumeDim, Frame::Inertial, DataVector>,
+    // derivatives of psi, pi, phi
+    ::Tags::Tempiaa<31, VolumeDim, Frame::Inertial, DataVector>,
+    ::Tags::Tempiaa<32, VolumeDim, Frame::Inertial, DataVector>,
+    ::Tags::Tempijaa<33, VolumeDim, Frame::Inertial, DataVector>,
+    // <34>, <35>, <36> defined above
+    // mapped coordinates
+    ::Tags::TempI<37, VolumeDim, Frame::Inertial, DataVector>>;
+
+// \brief This function computes intermediate variables needed for
+// Bjorhus-type constraint preserving boundary conditions for the
+// GeneralizedHarmonic system
+template <size_t VolumeDim, typename TagsList, typename DbTags,
+          typename VarsTagsList, typename DtVarsTagsList>
+void local_variables(
+    gsl::not_null<TempBuffer<TagsList>*> buffer, const db::DataBox<DbTags>& box,
+    const Direction<VolumeDim>& direction, const size_t& dimension,
+    const typename domain::Tags::Mesh<VolumeDim>::type& mesh,
+    const Variables<VarsTagsList>& /* vars */,
+    const Variables<DtVarsTagsList>& dt_vars,
+    const tnsr::i<DataVector, VolumeDim, Frame::Inertial>&
+        unit_interface_normal_one_form,
+    const typename Tags::CharacteristicSpeeds<VolumeDim, Frame::Inertial>::type&
+        char_speeds) noexcept {
+  //
+  // NOTE: variable names below closely follow \cite Lindblom2005qh
+  //
+  // 1) Extract quantities from databox that are needed to compute
+  // intermediate variables
+  using tags_needed_on_slice = tmpl::list<
+      gr::Tags::Lapse<DataVector>,
+      gr::Tags::Shift<VolumeDim, Frame::Inertial, DataVector>,
+      gr::Tags::SpatialMetric<VolumeDim, Frame::Inertial, DataVector>,
+      gr::Tags::InverseSpatialMetric<VolumeDim, Frame::Inertial, DataVector>,
+      gr::Tags::SpacetimeNormalOneForm<VolumeDim, Frame::Inertial, DataVector>,
+      gr::Tags::SpacetimeNormalVector<VolumeDim, Frame::Inertial, DataVector>,
+      gr::Tags::SpacetimeMetric<VolumeDim, Frame::Inertial, DataVector>,
+      gr::Tags::InverseSpacetimeMetric<VolumeDim, Frame::Inertial, DataVector>,
+      gr::Tags::ExtrinsicCurvature<VolumeDim, Frame::Inertial, DataVector>,
+      // ---- derivs of Psi, Pi, and Phi.
+      ::Tags::deriv<gr::Tags::SpacetimeMetric<VolumeDim, Frame::Inertial>,
+                    tmpl::size_t<VolumeDim>, Frame::Inertial>,
+      ::Tags::deriv<Tags::Pi<VolumeDim, Frame::Inertial>,
+                    tmpl::size_t<VolumeDim>, Frame::Inertial>,
+      ::Tags::deriv<Tags::Phi<VolumeDim, Frame::Inertial>,
+                    tmpl::size_t<VolumeDim>, Frame::Inertial>,
+      // ---- constraint damping parameters
```

Comment:
Please don't leave commented-out lines in a PR that is intended to be merged into develop. Instead, if you want to remove something the reader would expect to be there, a comment explaining why it's not there would be appropriate

Reply:
Removed these, sorry!